### PR TITLE
UG-586 Add Jira Notification for artefact builds

### PR DESCRIPTION
**Pull request by hughsaunders**
This builds uses Jira notification functions that must be merged first.

Depends on #183 
Connects rcbops/u-suk-dev#1664

### CODE REVIEW

**Comment by hughsaunders** [comment]
Lint job [test build](https://github.com/rcbops/rpc-gating/pull/188) succeeded, lets review/merge this one next.

File: rpc_jobs/rpc_artifact_build.yml [hunk 1]
```diff
@@ -121,26 +121,35 @@
...
+        try {{
+          node('ArtifactBuilder2') {{
+            dir("rpc-gating") {{
+                git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+                common = load 'pipeline_steps/common.groovy'
+                artifact_build = load 'pipeline_steps/artifact_build.groovy'
+            }}
+            artifact_build.apt()
+          }} // node ArtifactBuilder2
+            dir("rpc-gating") {{
```
Comment (odyssey4me) [inline rpc_jobs/rpc_artifact_build.yml]: Is the indentation right here? Does it matter? It looks like it should be two spaces to the left.

Comment (hughsaunders) [inline rpc_jobs/rpc_artifact_build.yml]: Thanks, fixed.

File: rpc_jobs/rpc_artifact_build.yml [hunk 2]
```diff
@@ -121,26 +121,35 @@
...
         // so that we can cater for builds on the more restricted images
         // used for artifact-based builds.
         common.prepareRpcGit("auto", env.WORKSPACE)
-        artifact_build.git()
-        artifact_build.python()
-        artifact_build.container()
+        try {{
+          node('ArtifactBuilder2') {{
+            dir("rpc-gating") {{
+                git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
```
Comment (hughsaunders) [inline rpc_jobs/rpc_artifact_build.yml]: Thanks, fixed.

**Pull request merged**